src/components/block/DonationProjectCard/DonationProjectCard.module.css

```diff
@@ -0,0 +1,30 @@
+.container {
+  height: 240px;
+  display: flex;
+  background-size: cover;
+}
+
+.bottombox {
+  display: flex;
+  flex-direction: row;
+  height: 72px;
+  align-self: flex-end;
+}
+
+.bar {
+  background: var(--color-sheltertech-blue);
+  height: 100%;
+  margin-right: 12px;
+  width: 8px;
+}
+
+.title {
+  color: var(--color-white);
```

Comment:
Let's put this color property on the `.container` selector, since it will be inherited by the children elements, so you can just set it in one place.

Reply:
👍 Removed color property from title and subtitle children elements and added a color property with the white value to the parent.